.gitignore

```diff
@@ -96,7 +96,6 @@ Backup[ ]of[ ]*.numbers/
 *.eps
 *.jpg
 *.jpeg
-*.png
```

Comment:
Hi @yizhou0522 can you add this line to the gitignore file again and do pull request again? I do not want to change the gitignore file